sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_v2/init.sql

```diff
@@ -0,0 +1,633 @@
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_experiment,
+    ARRAY_AGG(
+      environment.settings.attribution.medium IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_medium,
+    ARRAY_AGG(
+      environment.settings.attribution.source IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_source,
+    ARRAY_AGG(environment.system.apple_model_id IGNORE NULLS ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS apple_model_id,
+    ARRAY_AGG(metadata.geo.db_version IGNORE NULLS ORDER BY submission_timestamp ASC)[
+      SAFE_OFFSET(0)
+    ] AS db_version,
+    ARRAY_AGG(
+      payload.processes.parent.scalars.startup_profile_selection_reason IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS startup_profile_selection_reason,
+    ARRAY_AGG(
+      environment.settings.attribution.dltoken IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS download_token,
+    ARRAY_AGG(
+      environment.settings.attribution.dlsource IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS download_source
+  FROM
+    telemetry.new_profile
+  LEFT JOIN
+    UNNEST(environment.experiments) AS experiments
+  WHERE
+    DATE(submission_timestamp) >= starting_date
+  GROUP BY
+    client_id
+),
+shutdown_ping AS (
+  SELECT
+    client_id AS client_id,
+    MIN(submission_timestamp) AS submission_timestamp,
+    ARRAY_AGG(
+      environment.settings.attribution.campaign IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_campaign,
+    ARRAY_AGG(
+      environment.settings.attribution.content IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_content,
+    ARRAY_AGG(
+      environment.settings.attribution.experiment IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_experiment,
+    ARRAY_AGG(
+      environment.settings.attribution.medium IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_medium,
+    ARRAY_AGG(
+      environment.settings.attribution.source IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS attribution_source,
+    ARRAY_AGG(
+      environment.settings.attribution.dltoken IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS download_token,
+    ARRAY_AGG(
+      environment.settings.attribution.dlsource IGNORE NULLS
+      ORDER BY
+        submission_timestamp ASC
+    )[SAFE_OFFSET(0)] AS download_source
+  FROM
+    telemetry.first_shutdown
+  WHERE
+    DATE(submission_timestamp) >= starting_date
+  GROUP BY
+    client_id
+),
+main_ping AS (
+  SELECT
+    client_id AS client_id,
+    MIN(sample_id) AS sample_id,
```

Comment:
Using submission_timestamp_min to allow for comparing the 3 pings, which wouldn't return an exact result using submission_date. Thius field is [calculated in the source query](https://github.com/mozilla/bigquery-etl/blob/1418e2018b45f5536b0a797ffdadfaf8502a1bbf/sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql#L1166) as the the first timestamp of the day, [doesn’t return null values and matches the submission_date](https://sql.telemetry.mozilla.org/queries/92792/source)